GoogleTestAdapter/Core/Resources.resx

```diff
@@ -0,0 +1,444 @@
+  </data>
+  <data name="ParseDurationMessage" xml:space="preserve">
+    <value>Could not parse duration in line '{0}'</value>
+    <comment>{0} represents line</comment>
+  </data>
+  <data name="ParsePair" xml:space="preserve">
+    <value>Could not parse pair '{0}', exception message: {1}</value>
+    <comment>{0} is regex pair, {1} represents localized exception message (e.g. "Cannot load file")</comment>
+  </data>
+  <data name="ProcessExitCode" xml:space="preserve">
+    <value>Process has exited with code {0}</value>
+    <comment>{0} represents a number (e.g. 5)</comment>
+  </data>
+  <data name="ProcessFailed" xml:space="preserve">
+    <value>Could not list test cases of executable '{0}': executing process failed with code {1}</value>
+    <comment>{0} represents a unlocalized name of executable (e.g. cmd.exe) file, {1} means process exit code</comment>
+  </data>
+  <data name="ReadTestDurationError" xml:space="preserve">
+    <value>Could not read test durations: {0}</value>
+    <comment>{0} represents localized exception message (e.g. "Cannot load file")</comment>
+  </data>
+  <data name="Regex" xml:space="preserve">
+    <value>Regex '{0}'</value>
+    <comment>{0} represents unlocalized regex pattern used (e.g. ab*d+)</comment>
+  </data>
+  <data name="RegexParseError" xml:space="preserve">
+    <value>Regex '{0}' cannot be parsed: {1}</value>
+    <comment>{0} represents unlocalized regex pattern used (e.g. ab*d+), {1} represents localized exception message (e.g. "Cannot load file")</comment>
+  </data>
+  <data name="RegexTimedOut" xml:space="preserve">
+    <value>Regex '{0}' timed out: {1}</value>
+    <comment>{0} represents unlocalized regex pattern used (e.g. ab*d+), {1} represents localized exception message (e.g. "Cannot load file")</comment>
+  </data>
+  <data name="ReportedResultsToVS" xml:space="preserve">
+    <value>{0} Reported {1} test results to Visual Studio during test execution, executable: '{2}'</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5), {2} represents a unlocalized name of executable (e.g. cmd.exe) file</comment>
+  </data>
+  <data name="ReportedTestResults" xml:space="preserve">
+    <value>{0} Reported {1} test results to Visual Studio, executable: '{2}', duration: {3}</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5),{2} represents an unlocalized name of executable (e.g. cmd.exe) file, {3} means stopwatch time</comment>
+  </data>
+  <data name="Results" xml:space="preserve">
+    <value>Test {0}: Expected {1} ms, actual {2} ms</value>
+    <comment>{0} represents an unlocalized name for thread (e.g. "T001519"), {1} represents a number (e.g. 5), {2} represents a number (e.g. 5)</comment>
+  </data>
+  <data name="RunBatchException" xml:space="preserve">
+    <value>{0} {1} batch file caused exception, message: '{2}', executed command: '{3}'</value>
```

Comment:
added a separate BatchFile for "batch file" string. Fill in for {1} is already there {1} represents a localized name for test mode (e.g. "Test setup" or "Test teardown")